cloud/azure/bin/lib/bastion.sh

```diff
@@ -1,5 +1,15 @@
 #! /usr/bin/env bash
 
+#######################################
+# Helper function to get the formatted name of the
+# resource group, which is alphanumeric only
+# Arguments:
+#   1: the resource group name
+#######################################
+function get_formatted_name() {
+  echo "$(echo "${1:0:15}" | sed 's/[^[:alnum:]]//g')"
```

Comment:
```suggestion
  echo "${1:0:15}" | sed 's/[^[:alnum:]]//g'
```
Double check, but i think this will still work.

Reply:
Thanks! That seems to work!